### PR TITLE
[pt-BR] Replace @api/deki references

### PR DESCRIPTION
**Pull request by joaodinizgit**
Part of https://github.com/mdn/translated-content/issues/10487 for pt-BR.
(Second attempt)

Script used:

```
grep -riE "\(\/@api\/deki\/files\/[0-9]*\/=([^\)]+)\)" files/pt-br/ -l | xargs -I {} sed -i -E s/"\(\/@api\/deki\/files\/[0-9]*\/=([^\)]+)\)"/"(\L\1)"/ig {}
grep -riE "\/@api\/deki\/files\/[0-9]*\/=([^\)]+)\"" files/pt-br/ -l | xargs -I {} sed -i -E s/"\/@api\/deki\/files\/[0-9]*\/=([^\)]+)\""/"\L\1\""/g {}
```

## Files

- [x] console-dir.png
- [x] content_categories_venn.png
- [x] css_multibg.png
- [x] css_syntax_-_block.png
- [x] css_syntax_-_declaration.png
- [x] css_syntax_-_declarations_block.png
- [x] css_syntax_-_ruleset.png
- [x] css_syntax_-_statements_venn_diag.png
- [x] datalist.png
- [x] dimensions-offset.png
- [x] dom_ref_introduction_to_the_dom.gif
- [x] example.png
- [x] example1.png
- [x] example2.png
- [x] figure8.4.png
- [x] figure8.5.png
- [x] figure8.6.png
- [x] firefoxinnervsouterheight2.png
- [x] floats.png
- [x] google-rotated.png
- [x] html-dl1.png
- [x] html-dl2.png
- [x] html-dl3.png
- [x] htmlblinkelement.gif
- [x] meter1.png
- [x] meter2.png
- [x] mouseenter.png
- [x] mouseover.png
- [x] nesting.png
- [x] offsetleft.jpg
- [x] progress-1.png
- [x] progress-firefox.jpg
- [x] scrolltop.png
- [x] skewed-google.png
- [x] svg_overview.png
- [x] tabs_widget.png
- [x] text-overflow.png
- [x] timerresult.png

### CODE REVIEW

**Comment by joaodinizgit** [comment]
@caugner Same error

**Comment by caugner** [comment]
@joaodinizgit What happens if you run `yarn install && yarn fix:md`?

**Comment by joaodinizgit** [comment]
@caugner I have yarn installed v1.22.19. Which folder I have to run yarn fix:md? /translated-content or /translated-content/files/pt-br ?

**Comment by joaodinizgit** [comment]
@caugner I running `yarn fix:md `. It's checking all the files of all langs. I will wait.

**Comment by caugner** [comment]
Sounds good, it will finish eventually, and fix the errors. Then all you need to do is to commit and push those changes.

**Comment by caugner** [comment]
@joaodinizgit Thank you, the PR looks good.

Would you have time to go through the list of files (added as a checklist to the PR description) and check which one's exist in translated-content (next to the `index.md`) or in content (next to the corresponding `index.md`)? I should be able to provide most of the missing images from a backup, if necessary.

**Comment by joaodinizgit** [comment]
Hi @caugner, I checked the list and no files were found.

**Comment by caugner** [comment]
@joaodinizgit Please find the missing images in this ZIP file: [missing-images-pt-br.zip](https://github.com/mdn/translated-content/files/12883179/missing-images-pt-br.zip)

You will need to move each of them next to the `index.md` that references it. Let me know if you need any help. 🙏

**Comment by caugner** [comment]
@joaodinizgit Here are the remaining missing images (they had spaces instead of underscores in the backup): [missing-images-pt-br-2.zip](https://github.com/mdn/translated-content/files/12883910/missing-images-pt-br-2.zip)

**Comment by joaodinizgit** [comment]
@caugner Why these errors?

**Comment by caugner** [comment]
> @caugner Why these errors?

We have a `filecheck` (in [mdn/content](https://github.com/mdn/content/)) that checks if images can be optimized to be much smaller in file size. Since those images are rather old, they were never optimized, so that check complains. If you check out mdn/content next to translated-content, you can run `yarn install` in the content directory first and then you can execute [these `yarn filecheck` commands](https://github.com/mdn/translated-content/actions/runs/6497283197/job/17646035445?pr=16510#step:11:47) from the root of the content repository to resolve the errors.

**Comment by caugner** [comment]
@joaodinizgit Thank you so much. It looks like everything is done and ready for @josielrocha to review, right?

**Comment by joaodinizgit** [comment]
@caugner Now, it's done. Yes, only the review by Josiel.

File: files/pt-br/web/css/float/index.md [hunk 1]
```diff
@@ -92,15 +92,15 @@ h4 {
 </div>
 ```
 
-{{ EmbedLiveSample('Exemplos', '', '', '', 'Web/CSS/float') }}
+{{EmbedLiveSample('','400','180')}}
```
Comment (caugner) [inline files/pt-br/web/css/float/index.md:95]: This change seems to be unrelated?

File: files/pt-br/web/css/transform-function/rotate3d/index.md [hunk 2]
```diff
@@ -45,9 +45,9 @@ rotate3d(x, y, z, a)
     <tr>
       <th scope="col">Coordenadas cartesianas em ℝ<sup>3</sup></th>
       <td colspan="1">
-        <a href="/@api/deki/files/5987/=transform-functions-rotate3d_cart.png"
+        <a href="transform-functions-rotate3d_cart.png"
           ><img
-            src="/@api/deki/files/5987/=transform-functions-rotate3d_cart.png?size=webview"
+            src="transform-functions-rotate3d_cart.png?size=webview"
```
Comment (caugner) [inline files/pt-br/web/css/transform-function/rotate3d/index.md:50]:
```suggestion
            src="transform-functions-rotate3d_cart.png"
```

File: files/pt-br/web/css/transform-function/rotate3d/index.md [hunk 3]
```diff
@@ -90,9 +90,9 @@ rotate3d(x, y, z, a)
     <tr>
       <th scope="col">Coordenadas homogêneas em ℝℙ<sup>3</sup></th>
       <td colspan="1">
-        <a href="/@api/deki/files/5986/=transform-functions-rotate3d_hom4.png"
+        <a href="transform-functions-rotate3d_hom4.png"
           ><img
-            src="/@api/deki/files/5986/=transform-functions-rotate3d_hom4.png?size=webview"
+            src="transform-functions-rotate3d_hom4.png?size=webview"
```
Comment (caugner) [inline files/pt-br/web/css/transform-function/rotate3d/index.md:95]:
```suggestion
            src="transform-functions-rotate3d_hom4.png"
```

File: files/pt-br/web/html/content_categories/index.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ Há três tipos de categorias de conteúdo:
 - Categorias de conteúdo relacionado, as quais decrevem regras comuns de conteúdo para elementos relacioados;
 - Categorias de conteúdo específico, as quais descrevem categorias raras compartilhadas somente por poucos elementos, às vezes apenas em um contexto específico.
 
-[![Content_categories_venn.png](/@api/deki/files/6244/=Content_categories_venn.png?size=webview)](/@api/deki/files/6244/=Content_categories_venn.png)
+[![Content_categories_venn.png](content_categories_venn.png?size=webview)](content_categories_venn.png)
```
Comment (caugner) [inline files/pt-br/web/html/content_categories/index.md:14]:
```suggestion
[![Content_categories_venn.png](content_categories_venn.png)](content_categories_venn.png)
```

**Review by caugner — CHANGES_REQUESTED** [review]

File: files/pt-br/web/css/float/index.md (hunk 1 above)
Comment (joaodinizgit) [reply files/pt-br/web/css/float/index.md:95]: The old line was giving me error in PR-test, so I changed like found in US version page.

Comment (joaodinizgit) [inline files/pt-br/web/css/float/index.md:95]:
```suggestion
{{EmbedLiveSample('Exemplos','400','180')}}
```

**Review by caugner — APPROVED** [review]

**Review by josielrocha — APPROVED** [review]
LGTM ;)

**Comment by josielrocha** [comment]
Hey guys, thank you a lot!! 🚀
